samples/Cafe-Bot/MultiTurnPromptsBot.cs

```diff
@@ -69,10 +69,15 @@ public MultiTurnPromptsBot(BotAccessors accessors)
             _dialogs.Add(new TextPrompt("confirmBooking"));
             _dialogs.Add(new ConfirmPrompt("confirm"));
 
+            // The application values must match the ones during creation of the endpoint
             var applicationId = "cafebot";
-            var endpointKey = Keys.LanguageGenerationSubscriptionKey;
-            var endpointRegion = "westus"; // The region must be the subscription key's region.
-            _languageGenerationResolver = LanguageGenerationUtilities.CreateResolver(applicationId, endpointKey, endpointRegion);
+            var applicationRegion = "westus";
```

Comment:
same

Reply:
No, as these values should be explicitly set by the developer using the samples.